backend/build_docs.py

```diff
@@ -334,24 +307,88 @@ def get_opensta_docs() -> None:
         with open(save_path, 'wb+') as file:
             file.write(response.content)
         logging.debug('OpenSTA docs downloaded successfully.')
-        source_dict.update(track_src(f'{cur_dir}/data/pdf/OpenSTA'))
     else:
         logging.debug('Failed to download file. Status code:', response.status_code)
 
+    response = requests.get(opensta_readme_url)
+
+    save_path = 'data/markdown/OpenSTA_docs/OpenSTA_readme.md'
+    os.makedirs(os.path.dirname(save_path), exist_ok=True)
+
+    if response.status_code == 200:
+        with open(save_path, 'wb+') as file:
+            file.write(response.content)
+        logging.debug('OpenSTA readme downloaded successfully.')
+    else:
+        logging.debug('Failed to download file. Status code:', response.status_code)
+
+    track_src(f'{cur_dir}/data/markdown/OpenSTA_docs')
+    track_src(f'{cur_dir}/data/pdf/OpenSTA')
+
 
-def get_yosys_rtdocs() -> None:
-    logging.debug('Downloading Yosys RT docs...')
+def get_or_website_html() -> None:
+    logging.debug('Scraping OR website...')
     try:
         subprocess.run(
-            f'wget -r -A.html -P data/rtdocs {yosys_rtdocs_url} ',
+            f'wget -r -A.html -P data/html/or_website {or_website_url} ',
```

Comment:
Is the space needed at end of f-string?

Reply:
Instead of wget, is it possible to use `urllib.request`/ `requests`? It makes the code more readable + avoid running shell commands.

Reply:
https://stackoverflow.com/questions/24346872/python-equivalent-of-a-given-wget-command